File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    for i,j in zip(arr1, arr2):
        v12 = bin(i|j)[2:].zfill(n)
        v12 = v12.replace("1", "#")
        v12 = v12.replace("0", " ")
        answer.append(v12)
    return answer

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2) {
    let answer = new Array(n).fill("");

    for (let i = 0; i < n; i++){
        let a1V = arr1[i].toString(2).padStart(n, "0")
        let a2V = arr2[i].toString(2).padStart(n, "0")
        
        for (let j = 0; j < n; j++){
            (a1V[j] === "1" || a2V[j] === "1")? answer[i] += "#" : answer[i] += " "
        }

    }
    return answer;
}